app/scripts/components/common/map/controls/aoi/custom-aoi-control.tsx

```diff
@@ -249,19 +259,6 @@ function CustomAoI({
     mbDraw.trash();
   }, [features, aoiDeleteAll, map]);
 
-  let isAreaSelected = false;
-  let isPointSelected = false;
-
-  // @NOTE: map?._drawControl?.getSelected() needs access to mapboxgl draw context store,
-  // but the function gets called before mapboxdraw store is initialized (before being added to map) resulting in an error
-  // Wrapping with try block so the values get subbed even when the store is not available
-  try {
-    isAreaSelected = !!(map?._drawControl?.getSelected().features.length);
-    isPointSelected = !!(map?._drawControl?.getSelectedPoints()?.features.length);
-  } catch(e) {
-    isAreaSelected = false;
-    isPointSelected = false;
-  }
```

Comment:
Yes - but the reason that this could not be wrapped is addressed in this comment: https://github.com/NASA-IMPACT/veda-ui/pull/1251#discussion_r1837479489